tests/pass/intrinsics-x86-pause-without-sse2.rs

```diff
@@ -0,0 +1,26 @@
+// Ignore everything except x86 and x86_64
+// Any new targets that are added to CI should be ignored here.
+// (We cannot use `cfg`-based tricks here since the `target-feature` flags below only work on x86.)
+//@ignore-target-aarch64
+//@ignore-target-arm
+//@ignore-target-avr
+//@ignore-target-s390x
+//@ignore-target-thumbv7em
+//@ignore-target-wasm32
+// SSSE3 implicitly enables SSE3
+//@compile-flags: -C target-feature=-sse2
```

Comment:
What's the connection between the comment and the line that follows?

Reply:
copy-paste leftover